src/workflows-app/components/FilterableWorkflowTable.ts

```diff
@@ -151,6 +191,64 @@ const FilterableWorkflowTable = ({
     }
   };
 
+  const includeKey = useMemo(
```

Comment:
Nit. These variables contain arrays, but the name is singular and suggests a single value. Consider pluralizing the name.

```suggestion
  const includeKeys = useMemo(
```